docs/reference/known_limitations.md

```diff
@@ -15,25 +15,35 @@ We are developing a new feature that will allow you to modify these settings dir
 
 ## Backups
 
-The limitations related to backups in Percona Everest are as follows:
+Let's delve into the limitations of on-demand backups in Percona Everest. 
+
+### PostgreSQL limitations for on-demand backups
 
 - When attempting to delete a PostgreSQL database that contains backups created with Everest versions older than 1.0.0, the database may become stuck in the **Deleting** state. 
 
     **Workaround**: To prevent this, manually delete any backups created with versions prior to 1.0.0 by using the **Delete** action on the **Backups** page before deleting the database.
 
 - You cannot change the bucket name for a specific backup storage. Doing so will make any backups taken for that bucket unusable.
 
+- You can create on-demand backups using only the first three backup storages if there are more than three backup storages.
+
+    For example, if you create backup storages everest-1, everest-2, everest-3, and everest-4 sequentially, you can only use everest-1, everest-2, and everest-3 for on-demand backups.
```

Comment:
Maybe:

> You can create on-demand backups and schedules using only the first three backups storages if there are more than three backups storages in use across on-demand backups and schedules.
For example, if you create two schedules using backups storages everest-1 and everest-2 and an on-demand backup using backup storage everest-3, you can only use one of those storages to create an on-demand backups or a schedule."

Reply:
@fabio-silva I find this confusing. Could you please help me understand this.

Reply:
@fabio-silva WDYT about this:

If more than three backup storages are created, you can only use the first three storages across on-demand backups and schedules.

For example, if you have created two schedules using backup storage bucket-1 and bucket-2, and an on-demand backup using backup storage bucket-3, you can only utilize one of these three backup storages to create the next on-demand backup or a schedule.

Reply:
Fixed